src/XRP/issued-currency-client.ts

```diff
@@ -810,12 +814,14 @@ export default class IssuedCurrencyClient {
    * @param issuer The original issuer of the issued currency, encoded as an X-address (see https://xrpaddress.info/).
    * @param currency The three-letter currency code of the issued currency being redeemed.
    * @param amount The amount of issued currency to redeem.
+   * @param sendMaxValue The max amount to send.
    */
   public async redeemIssuedCurrency(
     sender: Wallet,
     currency: string,
     issuer: string,
     amount: string,
+    sendMaxValue?: string,
```

Comment:
When we were attempting to get transaction history for IOUs, we were getting an error specifying that we needed to provide a `sendMax`. From our slack convo, it sounds like this fix might already be included in another PR.

Reply:
this should already be fixed via another PR (already merged) so we definitely don't want this additional param here.